content/index.njk

```diff
@@ -1,7 +1,7 @@
 ---
 layout: base.njk
 title: Digital Society
-description: Landing page
+description: Digital Society, a non-profit cooperative building digital products for the social good home page
```

Comment:
nitpick: I prefer it without the "the", and "home page" is extraneous I think (or should at least have a divider to separate it from the rest)

```suggestion
description: Digital Society, a non-profit cooperative building digital products for social good | Home page
```